docs/commands/ts.createrule.md

```diff
@@ -67,6 +71,32 @@ is a `timeseries` type and is created before `TS.CREATERULE` is called.
 - The timestamp of a compacted sample added to the destination time series is set to the start timestamp the appropriate compaction bucket. For example, for a 10-minute compaction bucket with no alignment, the compacted samples timestamps are `x:00`, `x:10`, `x:20`, and so on.
 </note>
 
+## Examples
+
+<details open>
+<summary><b>Create a compaction rule</b></summary>
+
+Create a time series to store the temperatures measured in Tel Aviv.
+
+{{< highlight bash >}}
+127.0.0.1:6379> TS.CREATE temp:TLV LABELS type temp location TLV
+OK
+{{< / highlight >}}
+
+Next, create a compacted time series named `dailyAvgTemp` containing one compacted sample per 24 hours: the time-weighted average of all measurements taken from midnight to next midnight.
+
+{{< highlight bash >}}
+127.0.0.1:6379> TS.CREATE dailyAvgTemp:TLV LABELS type temp location TLV
+127.0.0.1:6379> TS.CREATERULE temp:TLV dailyAvgTemp:TLV AGGREGATION twa 86400000 
+{{< / highlight >}}
+
+Now, also create a compacted time series named _dailyDiffTemp_. This time series will contain one compacted sample per 24 hours: the difference between the minimum and the maximum temperature measured between 06:00 and 06:00 next day.
+
+{{< highlight bash >}}
+127.0.0.1:6379> TS.CREATE dailyDiffTemp:TLV LABELS type temp location TLV
+127.0.0.1:6379> TS.CREATERULE temp:TLV dailyDiffTemp:TLV AGGREGATION range 86400000 21600000
```

Comment:
Consider adding an explanation: 86400000 is the number of milliseconds in 24 hours, 21600000 is the number of milliseconds in 6 hours.